image.css

```diff
@@ -0,0 +1,208 @@
+body {
+  background: linear-gradient(to bottom, #66ffcc 0%, #66ccff 105%);
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+}
+
+header {
+  background-color: rgba(114, 195, 202, 0.575);
+  height: 80px;
+}
+
+a {
+  text-decoration: none;
+  color: #3d6dc5;
+}
+
+#item {
+  color: aliceblue;
+}
+
+@media screen and (max-width: 1129px) {
+  .content1 {
+    display: none;
+  }
+}
+
+.head {
+  display: flex;
+  padding: 32px 6px;
+}
+
+.title {
+  color: rgb(235, 114, 134);
+  text-decoration: none;
+}
+
+.content {
+  display: flex;
+  gap: 40px;
+  flex: 0 0 240px;
+  flex-wrap: wrap;
+  height: max-content;
+  margin-top: 50px;
+}
+
+.content1 {
+  margin-left: 115px;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
This class seems to be repeated in quite a few of your files, try to seperate the common properties